src/e4pkcclient.c

```diff
@@ -206,6 +233,11 @@ int e4c_unprotect_message(uint8_t *mptr,
     secs1970 = (uint64_t)time(NULL); /* this system has a RTC */
 #endif
     
+    storagecaps = e4c_get_storage_caps(storage);
+    if (!(storagecaps & E4_STORECAP_PUBKEY)) {
```

Comment:
Let's make this clearer and actually for a more grokable comparison since the result of this bitwise & is an *int*
```c
if ((storagecaps & E4_STORECAP_PUBKEY) == 0) {
```